docs/ref/request-response.txt

```diff
@@ -554,12 +554,11 @@ a subclass of dictionary. Exceptions are outlined here:
 
 .. method:: QueryDict.__getitem__(key)
 
-    Returns the value for the given key. If the key has more than one value,
-    it returns the last value. Raises
-    ``django.utils.datastructures.MultiValueDictKeyError`` if the key does not
-    exist. (This is a subclass of Python's standard :exc:`KeyError`, so you can
-    stick to catching ``KeyError``.)
-
+    Returns the last data value for the given key. If the key has more than one value,
+    it returns the last value. If the value is an empty list, ``[]`` is returned. 
+    Raises ``django.utils.datastructures.MultiValueDictKeyError`` if the key does not exist. 
+    (This is a subclass of Python's standard :exc:`KeyError`, so you can stick to catching ``KeyError``.)
```

Comment:
Hello! Docs should be wrapped at 80 characters: https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files